### PR TITLE
RM-121042 Release react-dart 6.1.5

### PR DESCRIPTION
**Pull request by greglittlefield-wf**

### CODE REVIEW

**Review by kealjones-wk — APPROVED** [review]
+1

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## [6.1.5](https://github.com/cleandart/react-dart/compare/6.1.4...6.1.5)##
+- [#333] Remove old, unused file from bin/
```
Comment (kealjones-wk) [inline CHANGELOG.md:2]: #nit 
```suggestion
- [#333] Remove old, unused file from bin.
```

**Review by kealjones-wk — APPROVED** [review]
#nit lol

**Comment by aviary-wf** [comment]
## Security Insights
<details>
<summary>(2) Vulnerable direct dependencies were detected</summary>
<li>1 vulns in <code>ansi-regex < 5.0.1</code> via <code>yarn.lock</code></li>
<li>1 vulns in <code>glob-parent < 5.1.2</code> via <code>yarn.lock</code></li>
</details>

## Action Items
* Review dependencies for available updates
* See <a href="https://splunk.workiva.net/en-US/app/search/security_vulnerable_dependencies?form.repository=react-dart">this Splunk dashboard</a> for more CVE details
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: CHANGELOG.md (hunk 1 above)
Comment (greglittlefield-wf) [reply CHANGELOG.md:2]: I actually did a slash on purpose to indicate it was a directory

**Comment by greglittlefield-wf** [comment]
QA +1

@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

File: CHANGELOG.md (hunk 1 above)
Comment (kealjones-wk) [reply CHANGELOG.md:2]: oh. well i did not get that.

**Pull request merged**